### DISCUSSION TITLE
Trying to upgrade from 3.6.40 to 4.11.5: login not working

### DISCUSSION THREAD

**Discussion by enribar** [globaleaks/globaleaks-whistleblowing-software#3493] [Q&A]
Hi, I am trying to upgrade my installation of globaleaks from version 3.6.40 to 4.11.5, moving it to a new Debian 11 server.
I followed the docs:

- create an archive backup of /var/globaleaks
- instantiate the lates Debian available
- log on the new server and extract the backup in /var/globaleaks
- follow the Installation Guide; GlobaLeaks while installing will recognize the presence of an existing data directory and will use it

On the new server globaleaks service starts, but login using the same users that on the old server work, always fail. I also tried resetting admin user password with `gl-admin resetpass admin`, but using new password does not make login work.
I tried searching for errors in globaleaks.log, but there are no lines referred to failed logins.

On Globaleaks.log I do not see any errors regarding upgrade, it correctly shows "Performing schema migration...", "Updating DB"...

Do you have any clue? Is there a way to debug this issue?

**Comment by evilaliv3** [comment]
Thank you @enribar for providing this feedback.

This will be soon fixed. Is a bug that we just spotted for very old setups like your. Users are correctly upgraded but the migrations is disabling them.

We will be issuing the fix withing days.

**Reply by enribar** [reply]
Hi, are there any updates about this fix? Thanks.

**Reply by evilaliv3** [reply]
It will be fixed by today or tomorrow with a release 4.12
